dashboard/test/ui/features/teacher-dashboard.feature

```diff
@@ -28,6 +31,31 @@ Feature: Using the teacher dashboard
     Then selector "a[href*='/s/allthethings/stage/2/puzzle/1']" has class "perfect"
     But selector "a[href*='/s/allthethings/stage/2/puzzle/2']" doesn't have class "perfect"
 
+    When I click selector "a:contains('Section: SectionName')" once I see it
+    And I click selector "#learn-tabs a:contains('Stats')" once I see it
+    And I wait until element "#stats-tab" is visible
+    And element "#stats-tab td:nth(0)" contains text "Sally"
+    And element "#stats-tab td:nth(1)" contains text "3"
+    And element "#stats-tab td:nth(2)" contains text "2"
+
+    When I click selector "#learn-tabs a:contains('Text Responses')" once I see it
```

Comment:
I can split each of these up into separate scenarios if you prefer.  I combined them because each `I create a teacher-associated student` step takes a fair amount of time.